### DISCUSSION TITLE
Full-text Search

### DISCUSSION THREAD

**Discussion by mgameover** [storybookjs/storybook#15692] [Help]
Hi guys,

I am wondering - Is there any way to customize/modify or even replace search engine that is being used in Storybook?

The thing is that we have a lot of docs (api description, usage guidelines and etc) that comes with our components (docs addon is used to display that).  It would be really great to do search not only by story's names but its documentation/description. My idea is to use search index instead e.g. [Lunr ](https://lunrjs.com/). 
But it looks there is no way to embed it in Storybook right now. 
Any thoughts?

Any help is highly appreciated. Thanks!

**Comment by bbellmyers** [comment]
see discussion https://github.com/storybookjs/storybook/discussions/24793
